stanford_profile_helper.module

```diff
@@ -24,6 +24,17 @@ use Drupal\node\NodeInterface;
 use Drupal\taxonomy\TermInterface;
 use Drupal\taxonomy_menu\Plugin\Menu\TaxonomyMenuMenuLink;
 
+/**
+ * Implements hook_theme().
+ */
+function stanford_profile_helper_theme($existing, $type, $theme, $path) {
+  $themes['block__stanford_basic_search'] = [
+    'template' => 'block--stanford-basic-search',
+    'original hook' => 'block',
+  ];
+  return $themes;
+}
```

Comment:
Why is this needed? Do the template suggestions not work here?

Reply:
not from within a module unfortunately.